### ISSUE TITLE
[bug] Start storybook when using the `test-runner`

### ISSUE THREAD

**Issue by alex-page** [storybookjs/test-runner#455]
**Describe the bug**
It's annoying to have to configure scripts to run storybook, wait for storybook to be loaded, run the tests. Storybook has a server and all the infrastructure to put these pieces together for me. Please add an option to start storybook server with the test runner.

**To Reproduce**
Create a test-runner test and you will have to configure your scripts to run, wait and then test.

**Expected behavior**
I expect the test runner for storybook to start a storybook server or have a flag to be able to do this.

**Comment by loganvolkers** [comment]
Luckily Playwright provides the [Web server](https://playwright.dev/docs/test-webserver) option to provide exactly this functionality.

**Comment by alex-page** [comment]
Yep. I am all for using the playwright web server to solve this. I have set this up on other projects and it is shocking to me this is locked away.
